src/main/java/seedu/ui/TextUi.java

```diff
@@ -56,9 +31,27 @@ public static void printBottomLine() {
         System.out.println("\n" + LINE);
     }
 
-    private static void initMessage() {
+    public static void welcomeMessage() {
+        printDoubleLineMessage(LOGO);
+        String message = "Welcome to ConTech, your personal contact tracker.\n"
+                + "Can I get your name?";
+        printBottomLineMessage(message);
+    }
+
+    public static void welcomeBackMessage(Contact personalContact) {
         printDoubleLineMessage(LOGO);
-        System.out.println("Welcome to ConTech, your personal contact tracker.\n" + LINE);
+        String message = "Hello, " + personalContact.getName() + ". Welcome back to ConTech, "
+                + "your personal \ncontact tracker.";
+        printBottomLineMessage(message);
+    }
```

Comment:
Noted that `welcomeBackMessage()` is for subsequent start up runs while `welcomeMessage()` and `greetingMessage()` are for first run (without a `me.txt` file).

Reply:
Good catch @lezongmun !